src/tiledb/cloud/client.py

```diff
@@ -63,6 +63,36 @@ def Ctx(config=None):
     return tiledb.Ctx(Config(config))
 
 
+class AWS(enum.Enum):
+    """
+    The AWS cloud regions that are currently supported.
+
+    US_EAST_1 = North America, United States (Virginia)
+    US_WEST_1 = North America, United States (California)
+    EU_WEST_1 = Europe, Ireland
+    EU_WEST_2 = Europe, London
+    AP_SOUTHEAST_1 = Asia, Singapore
+    """
+
+    US_EAST_1 = "https://us-east-1.aws.api.tiledb.com"
+    US_WEST_1 = "https://us-west-1.aws.api.tiledb.com"
+    EU_WEST_1 = "https://eu-west-1.aws.api.tiledb.com"
+    EU_WEST_2 = "https://eu-west-2.aws.api.tiledb.com"
+    AP_SOUTHEAST_1 = "https://ap-southeast-1.aws.api.tiledb.com"
+
+
+def login_to_region(region: AWS, **kwargs):
+    """
+    Login to cloud service, using the given region
+
+    :param region: the cloud region to run code in
+    :return:
+    """
+    if not region in AWS:
```

Comment:
I guess this should be `if region not in AWS` instead?

Reply:
They're semantically identical but your suggestions is more readable, so I addressed this in 5af2601.